entrypoint.sh

```diff
@@ -241,7 +250,10 @@ if [ -n "${LOGIN_USER}" ] && [ -n "${LOGIN_PASSWORD}" ]; then
 fi
 
 # tests the config file
-_echo "Testing compose file"
+_echo "Testing compose file:"
+echo "-----------------------------------"
+cat ${COMPOSE_FILE}
+echo "-----------------------------------"
```

Comment:
Same as above, please use `_echo`, so that it's clear that this is generated by the `entrypoint.sh` script.

Reply:
Done.